amundsen_application/static/js/components/common/Flag/index.tsx

```diff
@@ -32,14 +32,14 @@ const Flag: React.SFC<FlagProps> = ({ caseType, text, labelStyle }) => {
   // TODO: After upgrading to Bootstrap 4, this component should leverage badges
   // https://getbootstrap.com/docs/4.1/components/badge/
   return (
-    <span className={`flag label ${labelStyle}`}>{convertText(text, caseType)}</span>
+    <span className={`flag label label-${labelStyle}`}>{convertText(text, caseType)}</span>
```

Comment:
nit: I think there's value in expecting someone to enter `label-success` instead of `success`, because then our supported `labelStyles` can just match the existing bootstrap styles that everyone is familiar with. 

What I'm getting at is that someone coming in to use this component will have to remember that our supported styles don't need the `label-` prefix, so I think it might be more convenient to just match what we're leveraging.

Reply:
I was thinking we should abstract away the concept of the `label` and `label-{default | alert| etc}` classes so users of the Flag component don't care what the CSS classes are used under the hood. Also consider this will get converted to `badge badge-alert` in the future.

Reply:
I see. So this way when we switch to badges we won't have to update the `labelStyle` value on each usage, makes sense.